c2rust-transpile/src/c_ast/mod.rs

```diff
@@ -1720,6 +2004,24 @@ impl CTypeKind {
             Half => "half",
             BFloat16 => "bfloat16",
             Float128 => "__float128",
+
+            Int8 => "int8_t",
```

Comment:
Do we want to use the Rust primitives directly here? We can do it for all the fixed-size ones.

Reply:
No, this function is for printing types' names in C. It's not used as part of generating Rust, just for debugging.